include/deal.II/fe/fe_interface_values.h

```diff
@@ -235,6 +235,18 @@ class FEInterfaceValues
   std::vector<types::global_dof_index>
   get_interface_dof_indices() const;
 
+  /**
+   * Same as above but for level cells.
+   */
+  std::vector<types::global_dof_index>
+  get_mg_interface_dof_indices() const;
+
+  /**
+   * Get dof indices for generic loops.
```

Comment:
Can you expand what "generic loops" are? It's not a term we commonly use in deal.II (and in fact, I don't know what it means :-) )